yolox/evaluators/coco_evaluator.py

```diff
@@ -226,6 +238,20 @@ def convert_to_coco_format(self, outputs, info_imgs, ids):
                     "segmentation": [],
                 }  # COCO json format
                 data_list.append(pred_data)
+
+            image_wise_data.update({
```

Comment:
image_wise_data requires xyxy(xmin,ymin,xmax,ymax) format, but is currently feeding xywh(xmin,ymin,width,height) format due to `bboxes = xyxy2xywh(bboxes)` for coco.

therefore 

`image_wise_data.update{.....` 
should be should be moved to between the following lines
```
scores = output[:, 4] * output[:, 5]
for ind in range(bboxes.shape[0]):
```
while 
`bboxes = xyxy2xywh(bboxes)`
should be moved after `image_wise_data.update{.....`  and before 'for ind in range(bboxes.shape[0]):'


An alternative can be modifying logger.py arround the following code to reconvert xywh to xyxy.
will leave the decision to modify which depending on code clarity.

`                for i in range(len(prediction["bboxes"])):
                    bbox = prediction["bboxes"][i]
                    x0 = bbox[0]
                    y0 = bbox[1]
                    x1 = bbox[2]
                    y1 = bbox[3]
                    box = {
                        "position": {
                            "minX": min(x0, x1),
                            "minY": min(y0, y1),
                            "maxX": max(x0, x1),
                            "maxY": max(y0, y1)`


Anyway, the above modification solved the visualization bug in wandb browser.

Reply:
Thanks for the great pointer. I added this and it works like a charm!